gameinterface additions

INSANEMODE opened the pull request:
- add waittill_any_timeout overrideMethod to _integration_t6.gsc
- thread event handler calls in monitorEvents  in _integration_base.gsc

INSANEMODE committed:
-add waittill_any_timeout to _integration_t6.gsc
- thread event handler calls in monitorEvents

RaidMax commented on GameFiles/GameInterface/_integration_t6.gsc:
```diff
@@ -27,6 +27,7 @@ Setup()
     level.overrideMethods[level.commonFunctions.waittillNotifyOrTimeout] = ::WaitillNotifyOrTimeoutWrapper;
     level.overrideMethods[level.commonFunctions.isBot]                   = ::IsBotWrapper;
     level.overrideMethods[level.commonFunctions.getXuid]                 = ::GetXuidWrapper;
+    level.overrideMethods[level.commonFunctions.waitTillAnyTimeout]       = common_scripts\utility::waittill_any_timeout;
```
You'll need to use the wrapper pattern here (see IsBotWrapper and GetXuidWrapper).
Not all CoDs support the `call` keyword so to avoid that incompatibility I do not use it in the _base or _shared code.
As a result all function pointers must be to a local function.

INSANEMODE replied:
Addressed in b02a5f498f3eca88ebc9ebf60bfc459d0307b161
Also added to iw5, since the function also existed there.

Threading has been removed from event handlers, as well.

INSANEMODE committed:
- add WaitTillAnyTimeout to iw5
- remove unneeded thread from event handlers
- change WaitTillAnyTimeout in t6 to use a wrapper
